packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.test.tsx

```diff
@@ -97,6 +97,7 @@ describe('TooltipHost', () => {
       <TooltipHost
         calloutProps={calloutProps}
         content={content}
+        delay={TooltipDelay.zero}
```

Comment:
Modifying this test because before Callout was rendered no matter what delay you set.